xbmc/application/Application.cpp

```diff
@@ -577,18 +577,23 @@ bool CApplication::Initialize()
   // initialize (and update as needed) our databases
   CDatabaseManager &databaseManager = m_ServiceManager->GetDatabaseManager();
 
+  bool allDatabasesInitialized{false};
   CEvent event(true);
-  CServiceBroker::GetJobManager()->Submit([&databaseManager, &event]() {
-    databaseManager.Initialize();
-    event.Set();
-  });
+  CServiceBroker::GetJobManager()->Submit(
+      [&allDatabasesInitialized, &databaseManager, &event]()
+      {
+        allDatabasesInitialized = databaseManager.Initialize();
+        event.Set();
+      });
 
-  std::string localizedStr = g_localizeStrings.Get(24150);
+  const std::string connecting{g_localizeStrings.Get(24186)};
+  const std::string updating{g_localizeStrings.Get(24150)};
   int iDots = 1;
   while (!event.Wait(1000ms))
```

Comment:
@ksooo can this be read like
 "if the submitted event takes longer than 1000ms then display the text message on the splash"?

i'm asking because i'm testing LE13 on a slow raspberry pi and i _always_ see the message "Database migration in progress... please wait" ..... even though there is no database migration executed.
(i'm connecting to a shared mariadb)
in that case the amount of time should be increased a little, maybe 2500ms?

on my living room htpc - which is way faster - i do not get the message on the splash.